File: SocialPulse/UI/mobile_socialpulse/lib/models/recommend_result.dart
import 'package:json_annotation/json_annotation.dart';

part 'recommend_result.g.dart';

@JsonSerializable()
class RecommendResult {
  int? id;
  int? postId;
  int? firstCopostId;
  int? secondCopostId;
  int? thirdCopostId;

  RecommendResult(this.id, this.postId, this.firstCopostId, this.secondCopostId, this.thirdCopostId);

  factory RecommendResult.fromJson(Map<String, dynamic> json) =>
      _$RecommendResultFromJson(json);

  Map<String, dynamic> toJson() => _$RecommendResultToJson(this);
}

File: SocialPulse/SocialPulse.Core/Entities/RecommendResult.cs
namespace SocialPulse.Core
{
    public class RecommendResult : BaseEntity
    {
        public int PostId { get; set; }
        public int FirstCopostId { get; set; }
        public int SecondCopostId { get; set; }
        public int ThirdCopostId { get; set; }
    }
}
